htdocs/comm/propal/card.php

```diff
@@ -1628,7 +1628,7 @@
 	print '<td class="tdtop">' . $langs->trans('NotePublic') . '</td>';
 	print '<td valign="top">';
 	$note_public = $object->getDefaultCreateValueFor('note_public', (is_object($objectsrc)?$objectsrc->note_public:null));
-	$doleditor = new DolEditor('note_public', $note_public, '', 80, 'dolibarr_notes', 'In', 0, false, true, ROWS_3, '90%');
```

Comment:
The public note is rendered outside the backoffice and logged context. For example, into PDF or ODT documents.
For this reason, we should not add images where link is valid only inside a logged backoffice session.
This is why the "local explorer" is disabled for notes. The bug seems to be intothe page to edit "notes" where local explorer is enabled.
I think a better fix is to disable it also in the generic page to edit notes.So there is a consistency in behaviour and no features that lead to bugs with common use.
We can introduce a hidden option to enable the local browser, but it should be enabled in both create and edit and must be kept as hidden option because common end users can't understand why they can add an image that is not a public url and having it visible sometimes and not visible the other times.